docs/data-sources/connecting-data-sources.md

```diff
@@ -10,14 +10,15 @@ Connect your data sources with GlareDB and GlareDB Cloud.
 
 ## Connecting data sources using SQL
 
-Data sources can be added using [CREATE EXTERNAL TABLE] or
-[CREATE EXTERNAL DATABASE].
+Define new data sources with [CREATE EXTERNAL TABLE] or [CREATE
+EXTERNAL DATABASE]. You can add new data sources to a GlareDB instance
+at any time.
```

Comment:
Suggestion: "New data sources can be added to a GlareDB instance at any time."

It might be worth mentioning that it's also unlimited? (not sure)